book/website/reproducible-research/open/open-data.md

```diff
@@ -38,6 +38,15 @@ These are cultural challenges that might be addressed in changing practice going
 However, there are also legal, ethical or contractual reasons that sometimes prevent making data publicly available in its entirety or even in parts.
 Below, we discuss some reasons explaining why this may be the case.
 
+```{figure} ../../figures/data-privacy.jpg
+---
+height: 500px
+name: data-privacy
+alt: The image contains points about why private data may be used.
```

Comment:
Suggested alt text: An image detailing why private data should be used. A person stands next to a well with "private data" written on it and a padlock around it. It is black and white and blue. The text lists that "people deserve: dignity, agency, privacy, rights, confirmed consent."